### ISSUE TITLE
Change colour for professionnal training

### ISSUE THREAD

**Issue by cuchulainn-fun** [openfun/richie#2641]
## Feature Request

**Is your feature request related to a problem or unsupported use case? Please describe.**

Change of visual identity for professional training courses in the catalogue

![Image](https://github.com/user-attachments/assets/902dcd29-2853-406b-bdf7-c6cdc5fc49e2)

**Describe the solution you'd like**

- Change the colour of the label for the micro-certification category (https://www.fun-mooc.fr/fr/categories/types/formation-professionnelle/micro-certification/) that appears in the Richie CMS. It is currently red and should be changed to #088488

- Change the colour of the training thumbnails from #009FA1 to #088488

**Comment by kernicPanel** [comment]
WDYT about making the label background color matching the footer background ?

<img width="622" height="788" alt="Image" src="https://github.com/user-attachments/assets/66713229-fae7-4541-bb49-dd9e91e1f4fb" />

Can you provide a screenshot of a training thumbnail please ? I don't get what it is.

**Comment by NamFra** [comment]
@kernicPanel 

About the original color used in the footers of the training thumbnails:
- The COM team requested (on 18/03) that the color `#1BA0A8` be used for all references to professional training
- You can find the request here: https://github.com/openfun/fun-richie-site-factory/issues/263
- So, to change it everywhere, I think we just need to update [this line](https://github.com/openfun/fun-richie-site-factory/blob/fe9a4debcaed165b43620f152a02a15127caba9e/sites/funmooc/src/frontend/cunningham.cjs#L62) with the new color `#088488` provided by @cuchulainn-fun ? 🤞
